infra/release/release_note.md

```diff
@@ -1,28 +1,51 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- Add hide-and-show button
+- Show more files
+  - memory trace files
```

Comment:
Is the use of lowercase letters intentional?

Reply:
IMHO it's intentional. @dayo09 could you give me a comment?

Reply:
Well no.... it wasn't. May I change them?